go.sum

```diff
@@ -214,8 +214,10 @@ github.com/buger/jsonparser v0.0.0-20180808090653-f4dd9f5a6b44/go.mod h1:bbYlZJ7
 github.com/bugsnag/bugsnag-go v0.0.0-20141110184014-b1d153021fcd/go.mod h1:2oa8nejYd4cQ/b0hMIopN0lCRxU0bueqREvZLWFrtK8=
 github.com/bugsnag/osext v0.0.0-20130617224835-0dd3f918b21b/go.mod h1:obH5gd0BsqsP2LwDJ9aOkm/6J86V6lyAXCoQWGw3K50=
 github.com/bugsnag/panicwrap v0.0.0-20151223152923-e2c28503fcd0/go.mod h1:D/8v3kj0zr8ZAKg1AQ6crr+5VwKN5eIywRkfhyM/+dE=
-github.com/canonical/pebble v0.0.0-20221010231311-dfff36380603 h1:dNUWQ+B76pFtLM3cHfq8t+vEvQlsjEF8p7oZIgJu7Ek=
-github.com/canonical/pebble v0.0.0-20221010231311-dfff36380603/go.mod h1:qAIw8HY2rZZZ7QOhG9wD/4rtXvPJfvaOg+uWbhSABpc=
+github.com/canonical/pebble v0.0.0-20230127043025-054e5fbebc6b h1:pVYPfyf4CHvbUseS1B6Nbkv9RMkBeX95AZbB9pwswVM=
+github.com/canonical/pebble v0.0.0-20230127043025-054e5fbebc6b/go.mod h1:j3uyWpPkuWf8u0kB2v7n/XGVpYIg4luGetpSngCvzac=
+github.com/canonical/x-go v0.0.0-20230113154138-0ccdb0b57a43 h1:bey1JgA3D2EBabr2a7kWKj+JlEPxX1akv8rcRromotA=
```

Comment:
where did this come from?

Reply:
Because the strutil and randutil packages from Pebble are now in the shared `x-go`. See https://github.com/canonical/pebble/pull/177